packages/ferric/src/templates/Cargo.toml

```diff
@@ -11,7 +11,7 @@ crate-type = ["cdylib"]
 version = "3.1"
 default-features = false
 # see https://nodejs.org/api/n-api.html#node-api-version-matrix
-features = ["napi3"]
+features = ["napi4"]
```

Comment:
Last time I checked, this wasn't possible. If I remember correctly, it was due to the missing implementation of the Node-API threadsafe functions 🤔 But I might be wrong - that needs to be investigated for this to merge.

Reply:
Hmm.